README.md

```diff
@@ -103,6 +103,8 @@ GLOBAL OPTIONS:
    --grpc_port value             The port the EXPERIMENTAL gRPC server listens on. Set to 0 to disable. (default: 9092) [$BAZEL_REMOTE_GRPC_PORT]
    --profile_host value          A host address to listen on for profiling, if enabled by a valid --profile_port setting. (default: "127.0.0.1") [$BAZEL_REMOTE_PROFILE_HOST]
    --profile_port value          If a positive integer, serve /debug/pprof/* URLs from http://profile_host:profile_port. (default: 0, ie profiling disabled) [$BAZEL_REMOTE_PROFILE_PORT]
+   --read_timeout value          The HTTP read timeout for a client request in seconds. (default: 0s, ie disabled) [$BAZEL_REMOTE_READ_TIMEOUT]
+   --write_timeout value         The HTTP write timeout for a server response in seconds. (default: 0s, ie disabled) [$BAZEL_REMOTE_WRITE_TIMEOUT]
```

Comment:
It might be worth naming these flags in such a way that we can add other timeout values in the future (eg for proxy backends, and/or the profiling server) and still have consistent naming. And maybe explicitly mention in the description that these flags don't cover proxy backends.

Reply:
Agreed, adding more description and fixing names.